### DISCUSSION TITLE
Table 组件设置固定列宽，实际列宽与设置值有偏差

### DISCUSSION THREAD

**Discussion by Jiang2020426** [dotnetcore/BootstrapBlazor#3668] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

当Table组件显示的列比较少时，且设置了固定列宽时，且设置的列宽总和是大于Table总宽度（水平滚动条已出现）
8.4.1版本：demo中，设置的用户名Name，宽度为60，Table实际渲染宽度为60px（正确）
8.6.0版本：demo中，设置的用户名Name，宽度为60，Table实际渲染宽度为78px（有一定偏差）
![2](https://github.com/dotnetcore/BootstrapBlazor/assets/68992508/90c85331-1fce-43e7-a59c-cd488a5d1d50)
![1](https://github.com/dotnetcore/BootstrapBlazor/assets/68992508/983c846a-d1ab-4101-a5de-868c02af5f08)

![3](https://github.com/dotnetcore/BootstrapBlazor/assets/68992508/7962bf41-75a8-4db8-b429-259f1f8f55e2)

### Expected Behavior

8.6.0版本：当设置的总列宽大于Table总宽度，设置Name宽度为60，应该渲染宽度为60

### Steps To Reproduce

[BootstrapBlazorTable.zip](https://github.com/user-attachments/files/15816411/BootstrapBlazorTable.zip)
以上为2个版本的demo

### .NET Version

NET8

**Comment by bb-auto** [comment]
@Jiang2020426 Thank you for reporting. We will give triage later.
